oem.json

```diff
@@ -3149,37 +3304,83 @@
+        "description": "Needed for Moto Demo Mode\nenv = environment",
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.discovery",
+        "label": "Moto Discovery",
+        "description": "Not sure what it does",
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.dolby.dolbyui",
+        "label": "Dolby Atmos",
+        "description": "Dolby helps you rich and engaging sound.",
+        "removal": "replace"
+    },
+    {
+        "id": "com.motorola.dynamicvolume",
+        "label": "Adaptive volume",
+        "description": "Used to control the Multi-Volume feature, where apps can have their own volume level set. It can also automatically mute apps that you always mute manually.",
         "removal": "delete"
     },
     {
         "id": "com.motorola.easyprefix",
-        "description": "Motorola Easy Prefix (https://play.google.com/store/apps/details?id=com.motorola.easyprefix)\nAuto add CSP (Service Provider code) prefix to your phone when you're abroad.\nhttps://en.wikipedia.org/wiki/List_of_country_calling_codes\n\nThis seems to not work correctly and it's generally not a good idea to call home (via GSM) when you're abroad.\nIt's better and cheaper to use chat apps like Signal/Wire)",
+        "label": "Easy Prefix",
+        "description": "Auto add CSP (Service Provider code) prefix to your phone when you're abroad.\nhttps://en.wikipedia.org/wiki/List_of_country_calling_codes\nThis seems to not work correctly and it's generally not a good idea to call home (via GSM) when you're abroad.\nIt's better and cheaper to use chat apps like Signal/Wire",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.easyprefix"
+        ],
         "removal": "delete"
     },
     {
         "id": "com.motorola.email",
-        "description": "Motorola Easy Prefix (https://play.google.com/store/apps/details?id=com.motorola.easyprefix)\nAuto add CSP (Service Provider code) prefix to your phone when you're abroad.\nhttps://en.wikipedia.org/wiki/List_of_country_calling_codes\n\nThis seems to not work correctly and it's generally not a good idea to call home (via GSM) when you're abroad.\nIt's better and cheaper to use chat apps like Signal/Wire)\n\nMoto Email (https://play.google.com/store/apps/details?id=com.motorola.email)\n",
+        "description": "Auto add CSP (Service Provider code) prefix to your phone when you're abroad.\nhttps://en.wikipedia.org/wiki/List_of_country_calling_codes\nThis seems to not work correctly and it's generally not a good idea to call home (via GSM) when you're abroad.\nIt's better and cheaper to use chat apps like Signal/Wire",
+        "web": [
+            "https://play.google.com/store/apps/details?id=com.motorola.email"
+        ],
+        "removal": "delete"
+    },
+    {
+        "id": "com.motorola.enterprise.service",
+        "label": "Moto Thinkshield-MM",
+        "description": "Provides various security to Moto devices",
```

Comment:
It would be better if it contains our opinion about the capabilities of the app or any link.

Reply:
I think Moto thinkshield is a core feature that is provided with this stock rom.
I'm not sure the capabilities of this app since it has no activities/receivers/providers.
Permissions : REMOTE_ACCESS_PERMISSION
Generally I don't believe in extra protection/antivirus/malware programs in my opinion probably safe to disable. Will upload & link pithus report.